end-to-end-test/local/docker_compose/stop.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
Not sure what a lot of this logic does. Could be that a lot of this can be trimmed. All this is really doing is `docker-compose down`